### PR TITLE
Learning to Rank stabilization: Gsoc proposal

### PR DESCRIPTION
**Pull request by addy007-icy**

### CODE REVIEW

File: proposal.rst [hunk 1]
```diff
@@ -237,7 +261,115 @@ Project Timeline
...
+
+**Week 3: May 28 - June 2:**
+
+The next goal would be to adding a regression to combine multiple tests. 
+
+Expected deliverables: A base code for the regression problem to be written. 
+
+	1. Implement the feedforward backpropagation algorithm to combine various rankers by assignming random weights and then let them adjust according to the algorithm and learning rate. 
+
+**Week 4: May 28 - June 8:**
```
Comment (AyushP123) [inline proposal.rst]: Combine the goals of this week with the previous week .... as the time overlaps

File: proposal.rst [hunk 2]
```diff
@@ -237,7 +261,115 @@ Project Timeline
...
+Testing and benchmarking period.
+
+Expected Deliverables: Stability of xapian-letor stress tested with various dataset examples. 
+
+	1. Writing examples to test proper execution. 
+
+	2. Make sure xapian-letor is stable by reporting and fixing bugs if any.
+
+	3. Cleaning up any previous work. 
+
```
Comment (AyushP123) [inline proposal.rst:301]: Combine the first two weeks. Eliminate blank lines, like the couple below.

File: proposal.rst [hunk 3]
```diff
@@ -237,7 +261,115 @@ Project Timeline
...
+
+**Week 5:  June 8 - June 15:**
+
+Expected deliverables: All of the regression code ready. cleaned up, and mergable.
+
+	1. Complete and clean out the code.
+
+	2. Phase 1 evaluation reportable.
+
```
Comment (AyushP123) [inline proposal.rst]: Keep this as a buffer time, you might need that to catch up on whatever is pending. You don't need 1 week for evaluation. You are evaluated by the mentors on whatever goals you have completed. There will be at max 1 hour of discussion.

File: proposal.rst (hunk 2 above)
Comment (addy007-icy) [reply proposal.rst:301]: Did!

**Comment by ojwb** [comment]
Thanks.

The proposal deadline has now passed, and the purpose of this PR was to provide feedback on the proposal rather than actually as something to merge into the template, so closing.

**Pull request closed without merging**